pkg/jobs/jobspb/jobs.proto

```diff
@@ -77,6 +77,11 @@ message EncryptionInfo {
   map<string, bytes> encryptedDataKeyByKMSMasterKeyID = 3;
 }
 
+message StreamIngestionDetails {
+  // StreamAddress is the location of the stream which the ingestion job will
+  // read from.
+  string stream_address = 1;
```

Comment:
`casttype` to a `streamclient.StreamAddress`?

Reply:
done.